core/server/worker/src/main/java/alluxio/worker/block/BlockLockManager.java

```diff
@@ -111,7 +111,18 @@ public long lockBlock(long sessionId, long blockId, BlockLockType blockLockType)
       }
       lock = blockLock.writeLock();
     }
-    lock.lock();
+    try {
+      if (!lock.tryLock(1L, TimeUnit.MINUTES)) {
+        LOG.warn("tryLock {} block {} for session {} failed after 1 min, "
+                + "lock reference count = {}",
+            blockLockType, blockId, sessionId, blockLock.getReferenceCount());
+        lock.lock();
+      }
+    } catch (InterruptedException e) {
+      Thread.currentThread().interrupt();
+      lock.lock();
```

Comment:
If interrupted, do we still lock?
If tryLock failed, do we still lock?
@apc999 What's your thoughts?

Reply:
Can you help remove this tryLock changes in this PR for now? Other logics LGTM. 
I have a PR to move all lock.lock() to tryLock() https://github.com/Alluxio/alluxio/pull/12507
but still need more discussion with @apc999 to see if we want to move to tryLock

Reply:
Okay, It's make sense to use tryLock instead wait for lock forever with silence.